README.md

```diff
@@ -1083,6 +1120,15 @@ __AWESOME__ apps counter: __130__ 🎉
 - [x] [GitLab](https://gitlab.com/proninyaroslav/libretorrent)
 - [ ] Official page
 
+# TV Shows Tracker
+### Showly ❤️
+<img alt="Showly" height="64" src="https://camo.githubusercontent.com/3740231f58f9cd91dbf27285736f77ecfac4f63c02fceefcacbeb40778d3f896/68747470733a2f2f692e6962622e636f2f4368424e374c672f69632d6c61756e636865722e706e67">
+
+- [x] [Google Play](https://play.google.com/store/apps/details?id=com.michaldrabik.showly2&gl=US)
```

Comment:
not only that. `gl` fixes the link to that locale for time formats and units, so it would force imperial units and strange date formatting (what is 02-03-22 if you don't know it's US format?) onto everybody – as would `hl` fix the language :wink:

Reply:
I see and knew it forces link to appear "correctly" in US format, however Google's tracking techniques are enough in my opinion to track your country by your ip address and, then you have your desired format. So which option is better?
1. Force on each of those Google Play's links `&gl=US`.
2. Keep it clean, let it choose it for you.

I prefer the 2 option, but maybe you have any arguments to convince me otherwise?